MantidQt/CustomInterfaces/src/Reflectometry/ReflSettingsPresenter.cpp

```diff
@@ -292,25 +300,35 @@ void ReflSettingsPresenter::getExpDefaults() {
 
   auto cRho = inst->getStringParameter("crho");
   if (!cRho.empty())
-    defaults[2] = cRho[0];
+    defaults[2] = "\"" + cRho[0] + "\"";
 
   auto cAlpha = inst->getStringParameter("calpha");
   if (!cAlpha.empty())
-    defaults[3] = cAlpha[0];
+    defaults[3] = "\"" + cAlpha[0] + "\"";
 
   auto cAp = inst->getStringParameter("cAp");
   if (!cAp.empty())
-    defaults[4] = cAp[0];
+    defaults[4] = "\"" + cAp[0] + "\"";
 
   auto cPp = inst->getStringParameter("cPp");
   if (!cPp.empty())
-    defaults[5] = cPp[0];
+    defaults[5] = "\"" + cPp[0] + "\"";
 
   defaults[6] = alg->getPropertyValue("ScaleFactor");
 
   m_view->setExpDefaults(defaults);
```

Comment:
I think in `ReflSettingsPresenter::getExpDefaults()` you should not wrap the values in quotes, since it is not nice that they appear in the view. These properties are double array properties, to me it is not intuitive to see them wrapped. Wrapping should only happen behind the scenes, when setting the properties from the view to the algorithm (so in ` ReflSettingsPresenter::getReductionOptions()` as you did). For example, if one wants to set these settings manually (not getting the defaults), is he supposed to put them in quote marks?